packages/shared/src/recording/getRecordings.ts

```diff
@@ -273,3 +326,13 @@ export function getRecordings(processGroupIdFilter?: string): LocalRecording[] {
       .sort((a, b) => b.date.getTime() - a.date.getTime())
   );
 }
+
+function logNotFoundWarning(recordingId: string, entries: LogEntry[], entry: LogEntry) {
+  logger.error(`Recording with ID "${recordingId}" not found`, {
+    recordingId,
+    entry,
+    entries: JSON.stringify(
+      entries.filter(entry => entry.id === recordingId || entry.recordingId === recordingId)
+    ),
+  });
```

Comment:
we'd often log this multiple times which can be a little bit wasteful, maybe it would be better to aggregate those orphaned entries and log per recording id groups at the end?

Reply:
Yeah I'd briefly considered that and decided maybe it wasn't worth adding _state_ to improve the debug logging but lemme sit with the idea a bit more.

Reply:
i'm fine with keeping it this way but then perhaps we shouldn't log filtered `entries` here - they would just be scattered (but not duplicated) through those relevant logs